var/spack/repos/builtin/packages/r-cner/package.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RCner(RPackage):
+    """"Large-scale identification and advanced visualization of sets of
+        conserved noncoding elements."""
+
+    homepage = "http://www.example.co://bioconductor.org/packages/CNEr/"
+    url      = "https://git.bioconductor.org/packages/CNEr"
+
+    version('1.14.0', git='https://git.bioconductor.org/packages/CNEr', commit='b8634d65c51728c815127e22b45eba7c9b9db897')
+
```

Comment:
should this also have a depdency on `depends_on('r@3.4.0:3.4.9', when='@1.14.0')` ?

Reply:
The versioned dependency on r added now.